exercises/maxBy/maxBy.js

```diff
@@ -18,11 +18,24 @@
  */
 function maxBy(collection, predicate) {
   // This is your job. :)
+  let first = predicate(collection[0]);
+  let word = collection[0];
+  for (let item of collection) {
+    if (predicate(item) > first) {
```

Comment:
The value of `first` never changes, so this won't work. If you didn't notice this, reflect on the design of your sanity checks.

Consider this:

```javascript
function maxBy(collection, fn) {
  let result = collection[0];

  for (let item of collection) {
    if (fn(item) > fn(result)) {
      result = item;
    }
  }

  return result;
}
```